python/py_binary.bzl

```
@@ -23,10 +23,21 @@ load("//python/private/common:py_binary_macro_bazel.bzl", _starlark_py_binary =
 _py_binary_impl = _starlark_py_binary if config.enable_pystar else native.py_binary
 
 def py_binary(**attrs):
-    """See the Bazel core [py_binary](https://docs.bazel.build/versions/master/be/python.html#py_binary) documentation.
+    """Creates an executable Python program.
+
+    This is the public macro wrapping the underlying rule. Args are forwarded
+    on as-is unless otherwise specified. See
+    {bzl:obj}`py_binary <//python/private/common:py_binary_rule_bazel.bzl%py_binary>`
```

The link in the rendered docs just says 'py_binary', it would be better to be more verbose here and have something similar to the following for this and other macroed rules.

```suggestion
    the underlying {bzl:obj}`py_binary <//python/private/common:py_binary_rule_bazel.bzl%py_binary>` rule
```

---

Done